app/plugins/eu.esdihumboldt.hale.app.cli/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: hale CLI application
 Bundle-SymbolicName: eu.esdihumboldt.hale.app.cli;singleton:=true
-Bundle-Version: 5.0.0.qualifier
+Bundle-Version: 5.1.0.qualifier
```

Comment:
Here s well `5.0.1`. I believe it would be great to change this first in master using the groovy script and then merging to the release branch. 

I believe you can run the below script on master and then create a PR.

```
./updateversionnumbers.groovy -o 5.1.0 -n 5.0.1
```

Once the PR is merged to the master, then master can be merged to the release branch so that we have a well maintained commit history in master which then will be merged to the release branch. 

But we can also wait for @stempler's feedback if he has any other proposal.

Reply:
And than we need to change back the master to 5.1.0?
Why we just don't keep 5.0.1 as long as we have the next version and we change it when the release will arrive when we will know exactly the version number?